src/components/NFTArticle/SlateEditor/Elements/Blocks.tsx

```diff
@@ -316,10 +316,9 @@ export const BlockDefinitions: Record<EArticleBlocks, IArticleBlockDefinition> =
     icon: <i className="fa-solid fa-function" aria-hidden/>,
     buttonInstantiable: true,
     render: ({ attributes, element, children }) => (
-      <div className={style.article_wrapper_container}>
-        <BlockKatexEditor slateAttributes={attributes} slateElement={element}>
-          {children}
-        </BlockKatexEditor>
+      <div className={style.article_wrapper_container} {...attributes}>
+	{children}
```

Comment:
we must ensure that we render the `children` otherwise will throw an error when the selection is pointing to this block